packages/react-api/__tests__/api/stats.test.js

```diff
@@ -225,5 +225,20 @@ describe('stats', () => {
       expect(res).toEqual(TILESET_TEST.output);
       expect(mockedGetTileJson).toBeCalledWith({ source: TILESET_TEST.input.source });
     });
+
+    test('BigInt numbers are correctly parsed', async () => {
+      const jsonWithBigInt = '{"rows": [{"quadbin":5256683984082960383}]}';
+
+      const fetchMock = (global.fetch = jest.fn().mockImplementation(async () => {
+        return {
+          ok: true,
+          text: async () => jsonWithBigInt
+        };
+      }));
+
+      const res = await getStats({ source: QUERY_SOURCE, column: 'injuries' });
+
+      expect(res.rows[0].quadbin).toEqual(BigInt(5256683984082960383));
```

Comment:
Careful here! Writing this out as `BigInt(5256683984082960383)` will truncate the number before it's passed to the BigInt function, so these are not equal, note the `n` suffix:

```javascript
BigInt(5256683984082960383) === BigInt(5256683984082960383n) // false!
```